tests/test_wave_05.py

```diff
@@ -8,7 +8,7 @@ def test_clothing_has_default_category_and_to_str():
     cloth = Clothing()
     assert cloth.category == "Clothing"
     assert str(cloth) == "The finest clothing you could wear."
-
```

Comment:
Be a little careful making stray modifications to the tests. We can use git to recover unmodified versions if necessary. These modifications didn't impact the test, but I have seen students make a change to a test, forget they made the change, and then run into difficulty submitting since the tests that passed for them locally don't match the tests being run at submission.